lib/perl/Genome/Model/Tools/EpitopePrediction/NewPipeline.pm

```diff
@@ -0,0 +1,192 @@
+package Genome::Model::Tools::EpitopePrediction::NewPipeline;
+
+use strict;
+use warnings;
+use Genome;
+use File::Basename qw(dirname);
+use YAML;
+
+class Genome::Model::Tools::EpitopePrediction::NewPipeline {
+    is => 'Command::V2',
+    has_input => [
+        output_directory => {
+            is => 'Text',
+            doc => 'the directory where you want results stored',
+        },
+#should this be a build?
+        model => {
+            is => "Genome::Model::SomaticVariation",
+        },
+        alleles => {
+            is => 'Text',
+            doc => 'A list of allele names to be used for epitope prediction with NetMHC',
+            is_many => 1,
+        },
+        epitope_length => {
+            is => 'Text',
+            doc => 'Length of subpeptides to predict with NetMHC',
+        },
+        netmhc_version => {
+            is => 'Text',
+            doc => 'The NetMHC version to use',
+            valid_values => ['3.0','3.4'],
+            default_value => '3.4',
+        },
+        output_filter => {
+            is => 'Text',
+            doc =>
+                'Type of epitopes to report in the final output - select \'top\' to report the top epitopes in terms of fold changes,  \'all\' to report all predictions ',
+            valid_values => ['top', 'all'],
+        },
+    ],
+    has_calculated_optional => [
+        build => {
+            calculate_from => [qw(model)],
+            calculate => q/$model->last_succeeded_build/,
```

Comment:
Would it make sense to let the user specify a build directly?  If they gave a model ID they'd get the interactive prompt asking them to choose which build to use.